commander/commander.go

```diff
@@ -341,24 +355,27 @@ func (c *Commander) XClaim(result interface{}, streamName, groupName, consumerNa
 	for _, id := range idList {
 		cmd = cmd.Add(id)
 	}
-	if options.Idle != 0 {
-		cmd = cmd.Add("IDLE")
-		cmd = cmd.Add(options.Idle.Milliseconds())
-	}
-	if options.Time.Unix() > 0 {
-		cmd = cmd.Add("TIME")
-		cmd = cmd.Add(options.Time.Unix())
-	}
-	if options.RetryCount != 0 {
-		cmd = cmd.Add("RETRYCOUNT")
-		cmd = cmd.Add(options.RetryCount)
-	}
-	if options.Force {
-		cmd = cmd.Add("force")
-	}
-	if options.Justid {
-		cmd = cmd.Add("justid")
+	if options != nil {
+		if options.Idle != 0 {
+			cmd = cmd.Add("IDLE")
+			cmd = cmd.Add(options.Idle.Milliseconds())
+		}
+		if options.Time.Unix() > 0 {
+			cmd = cmd.Add("TIME")
+			cmd = cmd.Add(options.Time.Unix())
+		}
+		if options.RetryCount != 0 {
```

Comment:
Please check the inconsistencies for types, specifically `int`. and also check the inconsistencies for operators, specially `>` and `!=`.